tensorboard/data/server/pip_package/tensorboard_data_server/__init__.py

```diff
@@ -19,7 +19,7 @@
 
 # Version of this Python package. This may differ from the versions of
 # both TensorBoard and the data server.
-__version__ = "0.5.0"
+__version__ = "0.6.0-alpha.0"
```

Comment:
Can we change this to `0.6.0a0` to keep the consistent Python format?

I’m actually seeing that Python treats these as equivalent—

```python
>>> from pkg_resources import parse_version as pv
>>> pv("0.6.0a0") == pv("0.6.0-alpha.0")
True
```

—which is news to me. But this isn’t mentioned in PEP 440:

> The canonical public version identifiers MUST comply with the following scheme:
>
> ```
> [N!]N(.N)*[{a|b|rc}N][.postN][.devN]
> ```

So I think that I’d prefer to keep it strictly compliant, at least until
we can find some reference saying that the `-alpha.0` form is okay.

Reply:
Done.

Ah, nice catch, my original change was unintentional.  I also prefer the strict compliance with PEP 440.  I've force pushed and will do the rebase-and-merge once CI passes.

Thanks!